newIDE/app/src/AssetStore/AssetStoreContext.js

```diff
@@ -449,7 +449,7 @@ export const AssetStoreStateProvider = ({
 
   const publicAssetPacksByTag = React.useMemo(
     () => {
-      if (!publicAssetPacks) {
+      if (!publicAssetPacks || !publicAssetPacks.starterPacks) {
```

Comment:
sometimes the response doesn't contain `starterPacks` for some reason, this is a 2nd safe check to avoid calling a method on it afterwards if no results are here.